hooks/prepare-commit-msg

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+CHECKSTYLE="checkstyle-8.20"
+CHECKSTYLE_JAR="${CHECKSTYLE}-all.jar"
+ROOT_DIR=$(git rev-parse --show-toplevel)
+
+FILES=$(git diff --cached --name-only --diff-filter=ACM | cat)
+
+COMMIT_MSG="$(cat "$1")"
+
+# Skip check if the commit msg starts with "tmp"
```

Comment:
maybe put this in the readme? not a big deal though

Reply:
yup, I'll add a paragraph in the readme